src/qps.cpp

```diff
@@ -1009,7 +1009,7 @@ void Qps::make_command_menu()
     }
     connect(m_command, SIGNAL(triggered(QAction *)),
             SLOT(run_command(QAction *)));
-
+	    
```

Comment:
Please use a setting in your text editor so that trailing spaces are removed (also in your future patches).

Reply:
Ok.